samples/memfs/memfs.go

```diff
@@ -677,6 +681,20 @@ func (fs *memFS) OpenFile(
 		panic("Found non-file.")
 	}
 
+	if inode.name == CheckFileOpenFlagsFileName {
```

Comment:
Is it required to put this here into the sample file system, or could you instead add a unit test?

Ideally the sample file system wouldn’t contain any test code, only example code.

Reply:
Yeah I agree this isn't ideal, but I could not think of a way to unit test it without changing the logic of the sample file system itself. Otherwise the unit test cannot know the value for `OpenFlags`. Suggestions appreciated, or maybe we can remove this if you prefer manual testing of the value of `OpenFlags`.

Reply:
Nevermind, for some reason I thought the samples are separate from tests, but now I see that the only _test.go files are in the samples directory. That’s a cleanup for another time, then…